packages/core/src/Table/TableRow/TableRow.styles.js

```diff
@@ -0,0 +1,36 @@
+import { colors } from '@dhis2/ui-constants'
+import css from 'styled-jsx/css'
+
+export default css`
+    :global(tbody) > tr:hover > :global(td),
+    :global(tbody) > tr.isHoveringExpandedContent > :global(td) {
+        background-color: ${colors.blue050};
+    }
+    :global(tbody) > tr:hover > :global(th),
+    :global(tbody) > tr.isHoveringExpandedContent > :global(th) {
+        background-color: ${colors.grey300};
+    }
+    :global(tbody) > tr:active > :global(td) {
+        background-color: ${colors.blue100};
+    }
+    :global(tbody) > tr.selected > :global(td) {
+        background-color: ${colors.teal050};
+    }
+    :global(tbody) > tr.selected:hover > :global(td),
+    :global(tbody) > tr.selected.isHoveringExpandedContent > :global(td) {
+        background-color: ${colors.teal100};
+    }
+    :global(tbody) > tr.isExpanded > :global(td) {
+        border-bottom-color: transparent;
+    }
+    :global(tbody) > tr.isExpanded > :global(td.bordered) {
+        border-bottom-color: ${colors.grey300};
+    }
+    :global(tbody) > tr.draggable {
+        cursor: move;
+        user-select: none;
+    }
+    :global(tbody) > tr > :global(td.active) {
+        background-color: ${colors.blue100} !important;
+    }
+`
```

Comment:
This part also includes some styles for `td` elements. It could be that some of these can be removed. But one of the ideas behind this is also that even if a dev would place plain `td` elements in our `TableRow` they would get some functionality out of the box. Upon reflection I think I'd better check which of these styles we actually need and keep these td styles to a bare minimum.

Reply:
> keep these td styles to a bare minimum.

I agree with this. I wouldn't cover rare edge cases like putting a plain `td` inside the table.